### DISCUSSION TITLE
Typo in wiki for SX127x/RFM9x - FSK modem

### DISCUSSION THREAD

**Discussion by pete20211207** [jgromes/RadioLib#437] [General]
Guess
[https://github.com/jgromes/RadioLib/wiki/Default-configuration#sx127xrfm9x---fsk-modem](url)
needs a minor typo corretion:
`SX1278::beginFSK(434.0, 48.0, 125.0, 10, 16, false);`
shall be
`SX1278::beginFSK(434.0, 48.0, 50.0, 125.0, 10, 16, false);`
...the frequency deviation is missing in the command...

**Comment by jgromes** [comment]
You're right, thanks for pointing that out. The values were also from the pre-5.0 versions, at which point the default FSK changed to 4.8 kbps, 5.0 kHz, so I updated that as well. Should be all fixed now.
